lxd/device/disk.go

```diff
@@ -765,11 +772,11 @@ func (d *disk) Update(oldDevices deviceConfig.Devices, isRunning bool) error {
 func (d *disk) applyDeferredQuota() error {
 	v := d.volatileGet()
 	if v["apply_quota"] != "" {
```

Comment:
@stgraber using `shared.IsTrue(v["apply_quota"])` here would be a useful hint to the reader it doesn't container the new size(s) anymore.

Reply:
@stgraber although are you leaving this here to account for previously unactioned deferred size changes? If so, a little comment here to advise that to avoid future changes would be good.

Reply:
Yeah, was thinking of the unlikely case where LXD gets upgraded while some instances have it set already.